### DISCUSSION TITLE
subsai_vad.ipynb example: to manage all files in a folder

### DISCUSSION THREAD

**Discussion by dgoryeo** [absadiki/subsai#6] [Ideas]
Thanks for this great work. I was wondering if it would be possible to tweak the example of subsai_vad.ipynb to process all files in a folder. I think it is a usecase that may help many potential users.

Kind regards

**Comment by absadiki** [comment]
You are welcome @dgoryeo. 

Yeah good idea ..  I think it is possible, a  `for loop` over all the files inside that folder. 
But I don't think people are interested yet. 
It is a great idea though, I'll definitely add this feature in the next release if there are more people asking about it.
